### ISSUE TITLE
Fatal error: Cannot open temporary file '\s8so.' -- aborting

### ISSUE THREAD

**Issue by Mazdaywik** [open-watcom/open-watcom-v2#198]
If user haven't permission to write to disk root, this error is appearing. In Open Watcom 1.9 this problem isn't appear.

**Comment by jmalak** [comment]
Please, could you give here more info, what function you use for file name/open.
In OW 2.0 was done a few changes in C run-time library for better conformance with ISO/POSIX standard.
Anyway I thing it is correct behaviour if you want create file in root and you have no appropriate rights.

**Comment by Mazdaywik** [comment]
Reproducing this error:
File foo.cpp:
```c++
int foo(int x) {
  return x * 100;
}
```
File main.cpp:
```c++
#include <cstdio>

int foo(int x);

int main() {
  std::printf("foo(%d) = %d\n", 100, foo(100));
  return 0;
}
```

Output:
```
C:\Users\AKonovalov\Documents\watcom>\WATCOM2\owsetenv.bat
Open Watcom Build Environment

C:\Users\AKonovalov\Documents\watcom>cl foo.cpp main.cpp
Open Watcom C/C++ CL Clone for 386
Version 2.0 beta Apr  2 2015 10:29:02 (64-bit)
Copyright (c) 2002-2015 The Open Watcom Contributors. All Rights Reserved.
Portions Copyright (c) 1995-2002 Sybase, Inc. All Rights Reserved.
Source code is available under the Sybase Open Watcom Public License.
See http://www.openwatcom.org/ for details.
Compatible with CL Version 13.0.0
foo.cpp
main.cpp
Fatal error: Cannot open temporary file '\s8hk.' -- aborting
```

Watcom 2, 64-bit host compiler, installed in C:\WATCOM2

**Comment by jmalak** [comment]
Thanks for details.
It looks like problem with handling temporary file in OW clone for Microsoft CL command driver.
I will lokk on this problem and fix it ASAP.

**Comment by jmalak** [comment]
I did a small investigation.
Problem is with Microsoft implementation of tmpnam function, it is used for 64-bit version (OW is build by MS VC). If you try 32-bit version (it is build by OW) it should work (it is same as in OW 1.9).
Anyway I will change code for OW clones for MS command drivers to have similar behaviour for any 
comipiler tool chains.

**Comment by jmalak** [comment]
It is fixed in git repository by change 159331433c227b6e1019fcb4ba9a2d4659b27d86.
